compiler/rustc_hir_analysis/src/coherence/inherent_impls_overlap.rs

```diff
@@ -319,9 +319,7 @@ impl<'tcx> InherentOverlapChecker<'tcx> {
             // List of connected regions is built. Now, run the overlap check
             // for each pair of impl blocks in the same connected region.
             for region in connected_regions.into_iter().flatten() {
-                let mut impl_blocks =
-                    region.impl_blocks.into_iter().collect::<SmallVec<[usize; 8]>>();
-                impl_blocks.sort_unstable();
+                let impl_blocks = region.impl_blocks.into_iter().collect::<SmallVec<[usize; 8]>>();
```

Comment:
Than `into_iter().collect()` can actually be skipped?

Reply:
It might be possible to use the [`IndexSet::as_slice`](https://docs.rs/indexmap/latest/indexmap/set/struct.IndexSet.html#method.as_slice) API